packages/react-app/src/components/main-view.tsx

```diff
@@ -73,9 +75,18 @@ function MainView({ children, toggleTheme }: Props) {
   const [scrollTopState, setScrollTop] = useState(0);
   const filterRef = useRef<HTMLDivElement>(null);
   const scrollViewRef = useRef<HTMLDivElement>(null);
-  const { page } = usePageContext();
+  const { page, setPage } = usePageContext();
+  const { pathname } = useLocation();
 
   useEffect(() => {
+    if (pathname && pathname.length > 1) {
```

Comment:
Pathname will be the same for detail and home no ?

Reply:
![image](https://user-images.githubusercontent.com/13890618/162461337-65d96f6a-685d-4299-a1ad-c06d0d9687da.png)

Reply:
pathname from useLocation its not the same pathname, because it simulate routes.

Logging pathname
![image](https://user-images.githubusercontent.com/691510/162469493-60472ae9-8ba6-4e61-93a1-ea53ad8c1ecc.png)